doc/reference/plutus/reference-script-example.md

```diff
@@ -0,0 +1,146 @@
+# Plutus Scripts
+
+## What is a reference script?
+
+A reference script is a script that exists at a particular transaction output. It can be used to witness, for example, a UTxO at the corresponding script address of said reference script. Why is this useful? It means we no longer have to include the script in the transaction, greatly reducing its size
+
+### An example of using a Plutus V2 reference script
+
+Below is an example that shows how to use a Plutus spending script. We will walk though the [shell script example of how to use a reference script to spend a tx input](scripts/plutus/example-reference-script-usage.sh).This is a step-by-step process involving:
+
++ the creation of the `Required Redeemer` Plutus txin script
++ the creation of the `Required Redeemer` Plutus script at a transaction output (creation of the reference script)
++ sending ADA and a datum to the Plutus script address
```

Comment:
In our style guide, we always use 'ada' instead of 'ADA'. Just a suggestion to change this, but up to you @Jimbo4350